src/components/LiderbordCardComponent.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import Card from "@mui/material/Card";
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import Button from "@mui/material/Button";
+import Divider from "@mui/material/Divider";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Liderbord from "../model/liderbord";
+
+const bull = (
```

Comment:
maybe change the name to something like "bulletPoint" or "seperatorPoint",m it's hard to know what this with only the name